extensions/wikia/Captcha/CaptchaHooks.class.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Captcha;
+
+class Hooks {
+
+	protected static $handler;
+
+	/**
+	 * Get the global Captcha instance
+	 *
+	 * @return Handler
+	 */
+	static function getInstance() {
+		if ( !self::$handler ) {
+			self::$handler = new Handler();
```

Comment:
You have to specify it as `\Captcha\Handler()` because without the leading forward slash you'd be specifying `\Captcha\Captcha\Handler`.  If you do give the fully qualified name (`\Captcha\Handler()`) while in the `Captcha` namespace, its not wrong but PHPStorm complains that its unecessary.